abichecker/abichecker.py

```diff
@@ -514,6 +539,12 @@ def check_one_request(self, req):
 
         self.dblogger.request_id = req.reqid
 
+        self.staged_in = None
+        for r in req.reviews:
```

Comment:
I'm not happy with this one. We should reduce not increase our dependency on this review - and make 'staging project' a property of a request. The problem is that there is little I have to offer as alternative.

`staging/$TARGET/staging_project?requests=1` will list all requests, but to request it only makes sense if you can do it for multiple requests at once (or rely on internal caching of osclib)

Reply:
the bot is long running so it would have to prune the cache on each iteration. I could move the staged_in property into ReviewBot so it's at least in a central place and not hidden in abichecker. The implementation can then be changed once OBS has better APIs

Reply:
sounds like a plan